src/Database/PostUpdate.php

```diff
@@ -5,11 +5,9 @@
 namespace Friendica\Database;
 
 use Friendica\Core\Config;
-use Friendica\Database\DBM;
```

Comment:
Because `Friendica\Database\DBM` is in the same namespace as `Friendica\Database\PostUpdate`. As such, a disambiguating `use` statement isn't necessary.

When encountering a non-fully qualified class name, PHP will first try the current namespace (the one declared at the top of the file). This is why we have to put global classes like  `Exception` in a `use` statement, because otherwise if would look for a `Friendica\Database\Exception` for example.

Here, when PHP finds the `DBM` class name, it's trying to load the `Friendica\Database\DBM` class as per the namespace declaration, which happens to be correct.